src/main/java/net/neoforged/moddevgradle/internal/NoIdeIntegration.java

```diff
@@ -0,0 +1,18 @@
+package net.neoforged.moddevgradle.internal;
+
+import org.gradle.api.Project;
+import org.gradle.api.tasks.TaskProvider;
+
+/**
+ * This implementation of {@link IdeIntegration} is used when no IDE was detected to host Gradle.
+ */
+final class NoIdeIntegration extends IdeIntegration {
+    public NoIdeIntegration(Project project) {
+        super(project);
+    }
+
+    @Override
+    protected void registerProjectSyncTask(TaskProvider<?> task) {
+        // No IDE
```

Comment:
Why are you nooping this? `idePostSync` should not be run in this case anyway, and it might be useful to preserve the behaviour for our own tests.